docs/general/server/plugins/index.mdx

```diff
@@ -289,6 +289,14 @@ Plugin for Jellyfin that adds theme songs to movies and tv shows using ThemerrDB
 
 - [GitHub](https://github.com/LizardByte/themerr-jellyfin)
 
+#### TubeArchivistMetadata
```

Comment:
Please add new entries to the end of the list

Reply:
Sorry, I thought they were sorted in alphabetical order.